src/main/java/com/kodilla/ecommercee/exception/OrderNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.ecommercee.exception;
+
+public class OrderNotFoundException extends Exception {
```

Comment:
na razie nie widzę miejsca gdzie używasz tego wyjątku... to jest "na przyszłość" ?